scripts/get-model-data.py

```diff
@@ -207,12 +207,13 @@
             'supplierId',
             'supplierName',
             'submittedAt',
-            'status',
+            'status_briefs',
```

Comment:
Does this need to be `status_brief`, as per the `data_duplicate_suffix` above?

Reply:
This is briefs as per the model name in the join. Actually, I'm going to make that clearer.

Reply:
Done